src/components/ProblemModal/ProblemModal.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { Modal, Text } from '@mantine/core';
+import HelpGif from '../HelpGif';
+import ProblemText from '../ProblemText';
+
+type ProblemModalProps = {
+  opened: boolean;
+  setOpened: (value: React.SetStateAction<boolean>) => void;
+};
+
+const ProblemModal = ({ opened, setOpened }: ProblemModalProps) => (
+  <Modal
+    id='problem-popover'
+    opened={opened}
+    onClose={() => setOpened(false)}
+    title='Problem:'
+    size='75vw'
+    centered
+    classNames={{
+      title: 'font-bold',
+      body: 'space-y-4',
+    }}
+  >
+    <ProblemText
+      classNames={{ provider: 'text-sm' }}
+      relativeParentId='problem-popover-body'
+      tooltipOffset={10}
```

Comment:
@kand198 Should this tooltip offset be decreased? It looks a bit funny with the modal:
![image](https://user-images.githubusercontent.com/68877945/189257639-8fe6aa08-37df-42b9-be90-39b0fd381304.png)